prime-dotnet-webapi/Models/Addresses/Address.cs

```diff
@@ -9,7 +9,8 @@ public enum AddressType
     {
         Physical = 1,
         Mailing = 2,
-        Verified = 3
+        Verified = 3,
+        Additional = 4,
```

Comment:
We might want to use a more explicit name for these since additional is super vague and currently paper enrolment is the only place this feature is built for.